pkg/build/sbom.go

```diff
@@ -26,42 +26,40 @@ import (
 	"istio.io/release-builder/pkg/util"
 )
 
-// Sbom generates Software Bill Of Materials for istio repo in an SPDX readable format.
+const (
+	sourceSpdx    string = "istio-source.spdx"
```

Comment:
i was wondering if we should have version appended to the file name... looks like all other artifacts have this. @ericvn thoughts?

Reply:
Good idea, I'll add the version name to the file name.
spdx is create for build/project, i.e combined spdx is generated for all docker tars

Reply:
We don't version manifest.yaml or a few others fwiw. Its already in a per-version folder (https://gcsweb.istio.io/gcs/istio-release/releases/1.16.0).

Reply:
but I am fine with it as well

Reply:
updated, ptal